src/styles/stars.scss

```diff
@@ -0,0 +1,21 @@
+.stars {
+  display: flex;
+  flex-direction: row;
+  gap: 4px;
+  &__star {
+    width: 16px;
+    height: 16px;
+    background-image: url('/src/images/star.svg');
```

Comment:
The path to the star image should be relative to the compiled CSS file, not the SCSS file. If your build process places the compiled CSS in a different directory, you need to adjust the path accordingly. For example, if the CSS file is in the 'dist' directory, the path might be '../images/star.svg'.